components/TextField/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Platform, StyleSheet, Text, View, TextInput } from 'react-native';
+
+export default class TextField extends React.Component {
+  render() {
+    return (
+    <View style={styles.textField}>
+      {this.props.title && <Text style={styles.titleText}>{this.props.title}</Text>}
+      <TextInput underlineColorAndroid={'transparent'}
+                 multiline={this.props.large}
```

Comment:
Perhaps use a `numLines` prop instead of a `large` boolean? Gives us more flexibility later and avoids hardcoding line numbers.